src/valkey-benchmark.c

```diff
@@ -1617,6 +1627,43 @@ long long showThroughput(struct aeEventLoop *eventLoop, long long id, void *clie
     return SHOW_THROUGHPUT_INTERVAL;
 }
 
+char *generateFunctionScript(uint32_t num_functions, int with_keys) {
+    /* 64K buffer to hold script code */
+    const size_t buffer_len = 64 * 1024;
+    char *buffer = zmalloc(buffer_len);
+    memset(buffer, 0, buffer_len);
+
+    int written = snprintf(buffer, buffer_len, "#!lua name=benchlib\n");
+    while (num_functions > 0 && (buffer_len - written) > 0) {
+        assert(buffer_len - written > 0);
+        int n = 0;
+        if (with_keys) {
+            n = snprintf(buffer + written, buffer_len - written,
+                         "local function foo%u(keys, args)\nreturn keys[0]\nend\n",
```

Comment:
This script is quite trivial. It tests only the overhead of starting the script runtime, etc.

If we want to compare Lua5.x to LuaJIT at some point, we may want more fancy scripts such as the ones in this comment: https://github.com/valkey-io/valkey/issues/1229#issuecomment-2439984030. We can do that later I guess.